wherehows-backend/app/controllers/Application.java

```diff
@@ -76,4 +81,37 @@ public static Result index() {
   public static Result healthcheck() {
     return ok("GOOD");
   }
+
+  public static Result deploylibsbackend() {
+    String env = System.getenv("WHZ_APP_HOME");
+    String libpath = env + "/lib";
+    String commitfile = env + "/commit";
+    String libraries = "";
+    String commit = "";
+
+    try {
+      BufferedReader br = new BufferedReader(new FileReader(commitfile));
+      while ((commit = br.readLine()) != null){
+
```

Comment:
My bad , i was printing the lines on console so i had used the loop. I have removed it now- Thanks for catching this